test/common/unit/libraries/MessageLib.t.sol

```diff
@@ -104,7 +104,7 @@ contract TestMessageLibIdentities is Test {
     function testNotifyShareClass(
         uint64 poolId,
         bytes16 scId,
-        string calldata name,
+        bytes32 name,
```

Comment:
Good catch, I think it was due `Halmos`, right?

Reply:
Yeah although now we are only fuzzing 32 bytes names rather than 128 byte names

Reply:
The border case that we need to ensure is that a string of 129 chars is correctly clamped to 128. I think maybe we need special test cases for those